src/User/EmployeeList.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import styled from "styled-components";
+import { Link } from "react-router-dom";
+
+const StyledEmployeesList = styled.ul`
```

Comment:
I Think this is exactly the same as another ul/li combination could create a common component and import to both!

Reply:
I think that is in another branch so when I resolve the merge conflicts in a moment I can do that.